### ISSUE TITLE
NVDA doesn't narrate menu name(label) when user enters into menu

### ISSUE THREAD

**Issue by kolaps33** [nvaccess/nvda#11903]
### Steps to reproduce:
NVDA is on and running
1. navigate to the URL https://www.w3.org/TR/wai-aria-practices-1.1/examples/menu-button/menu-button-links.html
2. navigate to the WAI-ARIA Quick Links  menu button  
3. press enter on the button
4. observe narration

### Actual behavior:
NVDA narrates: "W3C Home Page  1 of 6"

### Expected behavior:
NVDA could narrate as well label of the menu. 
I would expect NVDA narrates menu label because JAWS and VoiceOver does it. 
If we add to the menu label some important information, then NVDA users cannot hear it. 

#### Windows version:
10.0.19042

#### Name and version of other software in use when reproducing the issue:
Chrome Version 87.0.4280.66 (Official Build) (64-bit)

#### Does the issue still occur after restarting your computer?
yes 

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No 

#### If addons are disabled, is your problem still occuring?
yes 

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
no

**Comment by Adriani90** [comment]
In Firefox this seems wo work as expected if I am not mistaken. I guess Chrome does not expose this properly.
Maybe @jcsteh will have some thoughts on this issue, not sure how to report it to Chromium because I cannot see if this is exposed correctly in their accessibility inspector.

**Comment by jcsteh** [comment]
The fact that it works in Firefox is due to a happy accident of events being fired in the wrong order. It probably won't work once we fix that bug.

Not reading menu labels in NVDA is currently intentional, as it results in extra verbosity which most often isn't useful to the user. For example, if you're on a menu bar and you press enter to open the File menu, hearing "File menu" is redundant and makes the user wait longer to hear the important information, which is the first menu item in the menu. Even this W3C example is similar. The user has just engaged the WAI-ARIA Quick Links menu button, so they know it's going to open the WAI-ARIA Quick Links menu.

**Comment by jcsteh** [comment]
One possibility is that NVDA could read the menu label if it is somehow different to whatever last had focus. I can't see a good use case for that, but if you have a compelling use case, that might help support such a request.

**Comment by Adriani90** [comment]
I didn't see any example in the wild with a menu label being different to the button which opens the menu itself, and the initial author didn't provide a plausible use case. I agree the behavior as it is now coresponds to what we have in desktop applications as well and it is intended. Closing as won't fix.
